scripts/util.sh

```diff
@@ -87,6 +88,7 @@ createKsApp() {
   ks generate pipeline pipeline
 
   ks generate katib katib
+  ks generate modeldb modeldb
```

Comment:
Per comment above I would prefer to make modeldb optional until we get some usage on it.

Reply:
Done